lib/dor/services/client/objects.rb

```diff
@@ -27,6 +27,19 @@ def publish(object:)
           true
         end
 
+        # Notify the external Goobi system for a new object that was registered in DOR
+        # @param object [String] the pid for the object
+        # @raise [UnexpectedResponse] when the response is not successful.
+        # @return [boolean] true on success
+        def notify_goobi(object:)
```

Comment:
Can you add a delegation from client.rb into here so downstream users can call this method on the client object without having to know about the underlying objects/files/etc. structure?

Reply:
done